lib/spack/docs/build_systems/makefilepackage.rst

```diff
@@ -92,9 +92,9 @@ there are any other variables you need to set, you can do this in the
 
 .. code-block:: python
 
-   def edit(self, spec, prefix):
-       env["PREFIX"] = prefix
-       env["BLASLIB"] = spec["blas"].libs.ld_flags
+   def setup_build_environment(self, env):
```

Comment:
Can you update the name of the method a couple of lines above too, and check if the `cbench` example is still ok?

Reply:
Done. Yes, it's still a good example.